arches_querysets/datatypes/resource_types.py

```diff
@@ -85,10 +84,18 @@ def get_related_resources(self, tile, value):
         if not value:
             return []
         related_resources = []
+
+        if not isinstance(tile, models.TileModel):
+            tile_model = models.TileModel.objects.get(pk=tile["tileid"])
+        else:
+            tile_model = tile
```

Comment:
We already have the tile information from elastic, and all we need below is the resource, so it would be more efficient to just fetch the resource (instead of fetching both the resource and the tile)

One way to do that would be:
```suggestion
        if not isinstance(tile, models.TileModel):
            tile = models.TileModel(**tile)
```